randomizer/Lists/CBLocations/JungleJapesCBLocations.py

```diff
@@ -560,7 +560,15 @@
         logic=lambda l: l.swim,
         locations=[[5, 1.2, 1099, 165, 1496], [5, 1.2, 2307, 165, 1895]],
     ),
-    ColoredBananaGroup(group=40, map_id=Maps.JapesMountain, name="Bunch on Diddy's switch", konglist=[Kongs.diddy], region=Regions.Mine, logic=lambda l: l.peanut, locations=[[5, 1.0, 485, 144, 122]]),
+    ColoredBananaGroup(
+        group=40,
+        map_id=Maps.JapesMountain,
+        name="Bunch on Diddy's switch",
+        konglist=[Kongs.diddy],
+        region=Regions.Mine,
+        logic=lambda l: l.peanut or l.advanced_platforming,
```

Comment:
No logic required. Diddy can jump over the gap where the plank would spawn. This isn't relevant in over 99.999999% of the cases, since entering tends to require peanuts, but that's the logic that's applied to the switch and the coin up there.